devtools/cli/src/main/java/io/quarkus/cli/build/GradleRunner.java

```diff
@@ -225,7 +225,9 @@ void setGradleProperties(ArrayDeque<String> args, boolean batchMode) {
         }
         args.add(registryClient.getRegistryClientProperty());
 
-        final String configFile = System.getProperty(RegistriesConfigLocator.CONFIG_FILE_PATH_PROPERTY);
+        final String configFile = registryClient.getConfigArg() == null
```

Comment:
Should RegistryClientMixin be checking/handling this config file property for all cases (not just Gradle?)

Reply:
It's not necessary for Maven. It could be for jbang.

Reply:
We should look at sorting this out after... either this property is around or it isn't...